<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="labelData">
      <property name="text">
       <string>Data:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="comboBoxData">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>384</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="2" colspan="2">
     <widget class="QLabel" name="labelTimestep">
      <property name="text">
       <string>Timestep:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QPushButton" name="pushButtonGenerate">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="maximumSize">
       <size>
        <width>192</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Generate</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="pushButtonColor">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>24</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>24</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="lineEditIsovalueValue">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>48</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="labelIsovalue">
      <property name="text">
       <string>Isovalue:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSlider" name="horizontalSliderIsovalue">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>384</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksBelow</enum>
      </property>
      <property name="tickInterval">
       <number>100</number>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QPushButton" name="pushButtonDelete">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4" colspan="2">
     <widget class="QSpinBox" name="spinBoxTimestep">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="3" rowspan="2" colspan="3">
     <widget class="QTableView" name="tableViewIsosurfaces">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>770</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionAdd"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionClip"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionAdd"/>
   <addaction name="actionClip"/>
  </widget>
  <action name="actionOpen">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/folder</normaloff>:/icons/folder</iconset>
   </property>
   <property name="text">
    <string>&amp;Open Directory...</string>
   </property>
   <property name="toolTip">
    <string>Open Directory</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="toolTip">
    <string>Quit application</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAdd">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/plus</normaloff>:/icons/plus</iconset>
   </property>
   <property name="text">
    <string>&amp;Add Column...</string>
   </property>
   <property name="toolTip">
    <string>Add column</string>
   </property>
  </action>
  <action name="actionClip">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/cropping-tool</normaloff>:/icons/cropping-tool</iconset>
   </property>
   <property name="text">
    <string>&amp;Clipping Controls...</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>380</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
